server/controllers/comment.js

```diff
@@ -0,0 +1,66 @@
+import commentRepo from '../repository/commentRepository';
+import { goodHttpResponse, badHttpResponse } from '../utilities/httpResponse';
+import articleRepo from '../repository/articleRepository';
+
+/**
+ * Comment Controller class
+ */
+class Comment {
+  /** Create a new comment
+   * @param {object} request Request Object
+   * @param {object} response Response Object
+   * @returns {object} Comment Object
+   */
+  static async createComment(request, response) {
+    try {
+      const { slug } = request.params;
+      const article = await articleRepo.getArticle(slug);
+      if (article === null) {
+        return badHttpResponse(response, 404, 'We could not find this article');
+      }
+
+      request.body.articleId = article.id;
+      const newComment = await commentRepo.createComment(request.body);
+      return goodHttpResponse(response, 201, 'Comment created', newComment);
+    } catch (error) {
```

Comment:
Probably not a good Idea to return a raw error straight to the client. We should talk about this